Update README.md (#14)
SoftwareAG/cumulocity-asset-properties-widget: README.md

# Cumulocity Asset Properties Widget Plugin [<img width="35" src="https://user-images.githubusercontent.com/32765455/211497905-561e9197-18b9-43d5-a023-071d3635f4eb.png"/>](https://github.com/SoftwareAG/cumulocity-asset-properties-widget/releases/download/sag-pkg-asset-properties-widget-1.0.0/sag-pkg-asset-properties-widget-1.0.0.zip)

This Asset Properties Widget is the Cumulocity module federation plugin created using c8ycli. This plugin can be used in Application Builder or Cockpit. It fetches Inventory data based on the Asset and displays the same in a widget.

> ### Compatible:

Reviewer: ```suggestion
### Compatibility:
```

| APPLICATION BUILDER | CUMULOCITY  | ASSET PROPERTIES WIDGET PLUGIN |
| ------------------- | ----------- | ------------------------------ |

> | 2.0.x               | >= 1018.0.0 | 1.0.0                          |

Reviewer 2: incorporated

> ## Installation

Reviewer 2: yes, once release is done I need to provide release path to download the zip

> ## Installation

Reviewer 2: release path added.

### To install the widget-plugin in the Custom Cockpit application, follow the steps below:

> 1.	Click on Add extension Packages button in Administration > Ecosystem > Extension, where you can add your widget zip file.

Reviewer 2: extension availability means the allpication availability for sub tenant, you will get more info click on question mark icon.

2.	If you already have a custom Cockpit application, navigate to its Details page (Administration > Ecosystem > Applications > Cockpit(Custom)) and then to the Plugins tab. Install the Asset properties widget plugin.  [Extensions](https://cumulocity.com/guides/users-guide/administration/#extensions).

> 3.	If you don’t have your Custom Cockpit application, navigate to Administration > Ecosystem > Applications and click Add application. In the resulting dialog, select the option Duplicate existing application. From the list of applications select Cockpit (Subscribed). Use the default values and proceed once its done follow the **2nd steps**.
> 4. Now your widget is available in your Cockpit application. Navigate to the dashboard where the newly added widget is available in the list of widgets to add.

Reviewer: Remove these lines and add a note instead
```suggestion
Note: You need a custom application with Cumulocity IoT dashboard to see the widget in action. A quick way to achieve this is by creating a custom Cockpit application.
To create  a custom Cockpit application, follow the steps below:
1. Navigate to Administration > Ecosystem > Applications 
2. click **Add application**. 
3. In the resulting dialog, select the option **Duplicate existing application**. 
4. From the list of applications, select Cockpit (Subscribed). Use the default values and proceed.
```

## Installation

To install the Asset properties 2.0 widget plugin, you must add the plugin package to the extensions first and then install it on the desired custom application.

#### To add the plugin package to extensions

1.  Navigate to Administration > Ecosystem > Extensions
2.  Click **Add extension package** to upload the zip file

#### To install the widget plugin on a custom application

1. Navigate Adminstration > Ecosystem > Applications
2. Click application name to view the application details.
3. Switch to **Plugins** tab, Click **Install plugins**
4. In the resulting dialog, search for Asset properties 2.0 widget, click **Install**

To know more about extensions, see [Administration> Managing applications > Extensions](https://cumulocity.com/guides/users-guide/administration/#extensions) in _User guide_

> Note: You need a custom application with Cumulocity IoT dashboard to see the widget in action. A quick way to achieve this is by creating a custom Cockpit application.

Reviewer: ```suggestion

Note: You need a custom application with Cumulocity IoT dashboard to see the widget in action. A quick way to achieve this is by creating a custom Cockpit application.
```
